boards/arm/nrf5340_dk_nrf5340/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2019 Nordic Semiconductor ASA.
 # SPDX-License-Identifier: Apache-2.0
 
-if (CONFIG_BOARD_NRF5340_DK_NRF5340_CPUAPP)
+if (CONFIG_BOARD_NRF5340_DK_NRF5340_CPUAPP AND CONFIG_BOARD_ENABLE_CPUNET)
```

Comment:
Consider adding this library for ```BOARD_NRF5340_DK_NRF5340_CPUAPPNS``` target as well

Reply:
added